index.html

```diff
@@ -3007,69 +3007,299 @@ <h3>Truncating or limiting the length of strings</h3>
 	<p class="advisement">Specifications SHOULD NOT limit the size of data fields unless there is a specific practical or technical limitation.</p>
 	</div>
 
-    <p>There are many reasons why a length limit might be needed in a specification or format. Generally length limits correspond to underlying limits in the implementation, such as the use of fixed-size fields in a database or data store, the desire to fit into practical boundaries such as packet size, or some other implementation detail related to storage allocation or efficiency.</p>
-
+    <p>There are many reasons why a length limit might be needed in a specification or format. The most common reason is because there is an underlying limit for storage or interchange, such as the use of a fixed-sized field in a database, the desire to fit into practical boundaries such as packet size, or some other implementation detail related to storage allocation or efficiency. The other common reason is that there might be a limit on the size of the display or in the visible output.</p>
+    
     <p>When truncating strings, it's necessary to decide what units to use when counting the size of the string. In many cases this is beyond the control of the specification, since the truncation is occuring for some preordained reason. However, when the choice is available, some general guidelines can be applied.</p>
+    
+    <div class="req" id="char-trunc-avoid-visual-units">
+        <p class="advisement">Avoid specifying data field length limits in terms of [=visual text units=] (such as [=grapheme clusters=]). Use mechanisms such as CSS <code>text-overflow</code> that depend on font metrics instead.</p>
```

Comment:
I don't think`text-overflow` has anything to do with font metrics.  It merely describes what happens to text that doesn't fit into the space available.

Reply:
https://drafts.csswg.org/css-overflow/#text-overflow

Yeah, I think font metrics is only one factor, and there are many other factors such as letter/word spacing etc.